.github/CODEOWNERS

```diff
@@ -68,6 +68,7 @@
 
 /apps/alerting/ @grafana/alerting-backend @grafana/alerting-frontend
 /apps/playlist/ @grafana/grafana-app-platform-squad
+/apps/investigation/ @grafana/grafana-app-platform-squad
```

Comment:
@fcjack I've put this is as a placeholder – feel free to change this to the correct team!

Reply:
For now will be me and @matryer , until be ready for the team to assume.